config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  devise_for :users, path: '', path_names: {
+    sign_in: 'login',
+    sign_out: 'logout',
+    registration: 'signup'
+  },
+  controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+  
+  resources :users
+  resources :bookings
+  resources :packages
+
```

Comment:
You ought to only define routes that are used. see how here: https://nelsonfigueroa.dev/restricting-ruby-on-rails-routes-with-only-and-except/